Failed to docker:build for "karaf/camel-amq" quickstart

The steps to reproduce as below:

1. git clone https://github.com/fabric8io/ipaas-quickstarts
2. cd ipaas-quickstarts/quickstart/karaf/camel-amq
3. mvn clean install docker:build

<-----snip------>
May 03, 2016 6:50:35 AM org.apache.karaf.profile.assembly.AssemblyDeployCallback installFeature
INFO: Installing feature config for activemq-client/5.12.1
May 03, 2016 6:50:35 AM org.apache.karaf.profile.assembly.Builder bootStage
INFO: Feature 2e585d99-c0d1-44f3-99fc-8cbf4d6fdf2d is defined as a boot feature
May 03, 2016 6:50:35 AM org.apache.karaf.profile.assembly.Builder installArtifact
INFO: == Installing artifact mvn:org.apache.camel/camel-blueprint/2.17.0
[INFO] 
[INFO] --- maven-surefire-plugin:2.19.1:test (int-test) @ karaf-camel-amq ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ karaf-camel-amq ---
[INFO] No primary artifact to install, installing attached artifacts instead.
[INFO] Installing /root/ipaas-quickstarts/quickstart/karaf/camel-amq/pom.xml to /root/.m2/repository/io/fabric8/quickstarts/karaf-camel-amq/2.3-SNAPSHOT/karaf-camel-amq-2.3-SNAPSHOT.pom
[INFO] Installing /root/ipaas-quickstarts/quickstart/karaf/camel-amq/target/classes/kubernetes.json to /root/.m2/repository/io/fabric8/quickstarts/karaf-camel-amq/2.3-SNAPSHOT/karaf-camel-amq-2.3-SNAPSHOT-kubernetes.json
[INFO] Installing /root/ipaas-quickstarts/quickstart/karaf/camel-amq/target/classes/kubernetes.yml to /root/.m2/repository/io/fabric8/quickstarts/karaf-camel-amq/2.3-SNAPSHOT/karaf-camel-amq-2.3-SNAPSHOT-kubernetes.yml
[INFO] Installing /root/ipaas-quickstarts/quickstart/karaf/camel-amq/target/karaf-camel-amq-2.3-SNAPSHOT.zip to /root/.m2/repository/io/fabric8/quickstarts/karaf-camel-amq/2.3-SNAPSHOT/karaf-camel-amq-2.3-SNAPSHOT.zip
[INFO] 
[INFO] --- maven-bundle-plugin:2.3.7:install (default-install) @ karaf-camel-amq ---
[ERROR] file doesn't exist: file:/root/.m2/repository/io/fabric8/quickstarts/karaf-camel-amq/2.3-SNAPSHOT/karaf-camel-amq-2.3-SNAPSHOT.jar
[INFO] Writing OBR metadata
[INFO] 
[INFO] --- docker-maven-plugin:0.14.2:build (default-cli) @ karaf-camel-amq ---
[WARNING] The assembly descriptor contains a filesystem-root relative reference, which is not cross platform compatible /
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.703s
[INFO] Finished at: Tue May 03 06:50:37 EDT 2016
[INFO] Final Memory: 56M/344M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.14.2:build (default-cli) on project karaf-camel-amq: Failed to create assembly for docker image: Error adding file to archive: /root/ipaas-quickstarts/quickstart/karaf/camel-amq/target/karaf-camel-amq-2.3-SNAPSHOT.tar.gz with mode dir -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

--- a/quickstart/karaf/camel-amq/pom.xml
+++ b/quickstart/karaf/camel-amq/pom.xml
@@ -295,7 +295,7 @@
                     <files>
                       <!-- copy karaf archive -->
                       <file>
-                        <source>${project.build.directory}/${project.artifactId}-${project.version}.tar.gz</source>
+                        <source>${project.build.directory}/${project.artifactId}-${project.version}.zip</source>
                         <outputDirectory>/</outputDirectory>
                         <fileMode>0644</fileMode>
                       </file>

